src/core/layout/qgslayoutexporter.h

```diff
@@ -388,6 +388,12 @@ class CORE_EXPORT QgsLayoutExporter
        */
       QVector<qreal> predefinedMapScales;
 
+      /**
+       * Whether to query legend graphics synchronously. Necessary for WMS
+       * GetPrint, as otherwise processing the request ends before remote
+       * graphics are downloaded.
+       */
+      bool synchronousLegendGraphics = false;
```

Comment:
When would we ever want this to be false? Shouldn't we just make it always on for layout exports?